test/test_sparse.py

```diff
@@ -82,6 +84,59 @@ def ignore_op(func):
             torch.ops.aten._values.default,
         )
 
+class TestSparse000(TestCase):
+    # we want to run the tests below once before all other test cases,
```

Comment:
I don't think we can guarantee this order on some internal systems. I assume that's only important for development of this PR? If so can we change the name to something more expressive?

Reply:
This test only works as intended when it is run before other tests that call legacy constructors because of the warn-once nature. That said, I now realize that other tests should never call the legacy constructors anyway, and if these do, these should be fixed. In fact, the test would be even stronger when the test is run after all other tests, although, I think we can consider it unnecessary.

In sum, I'll just rename the test class to TestSparseLegacyConstructors.

Reply:
Fixed.